packages/thunderstore-api/package.json

```diff
@@ -11,8 +11,19 @@
   ],
   "scripts": {
     "build": "tsc",
-    "dev": "tsc --watch"
+    "dev": "tsc --watch",
+    "test": "jest"
```

Comment:
Could this also be `jest --watch`? It would be nice to have this running in the background and the tests breaking as soon as something is up, instead of at the branch pushing or commit time.